src/clientManager.js

```diff
@@ -45,7 +46,15 @@ ClientManager.prototype.loadUsers = function () {
 		);
 	}
 
-	users.forEach((name) => this.loadUser(name));
+	// This callback is used by Auth plugins to load users they deem acceptable
+	const callbackLoadUser = (user) => {
+		this.loadUser(user);
+	};
+
+	if (!Auth.loadUsers(users, callbackLoadUser)) {
+		// Fallback to loading all users
```

Comment:
I don't think this fallback is needed, just make the local auth return true (call the callback).

Reply:
A simpleLdapLoadUsers would also need to duplicate the same (albeit simple) code, and if other auth mechanisms were to be introduced in the future, they would also need to duplicate code. 

Having the fallback mechanism keeps the code in just the one place, close to where the function it calls is defined.

Reply:
Okay that's fair.